npc/custom/itembind.txt

```diff
@@ -24,8 +24,8 @@ prontera,144,174,4	script	Bound Items	4_M_JP_MID,{
 			close;
 		}
 		mes "What kind of bind?";
-		.@boundtype = 1 << (select("Account", "Guild", "Character")-1);
-		if(.@boundtype == 2 && (!getcharid(CHAR_ID_GUILD) || getguildinfo(GUILDINFO_MASTER_NAME, getcharid(CHAR_ID_GUILD)) != strcharinfo(PC_NAME))) {
+		.@boundtype = IBT_ACCOUNT << (select("Account", "Guild", "Character")-1);
```

Comment:
```suggestion
		.@boundtype = 1<< (select("Account", "Guild", "Character")-1);
```
Here, the 1 is not related to showing IBT_ACCOUNT, but rather showing it's bitwise, wherein 1 is left-shifted

Reply:
ah, it was wrong and not intended, forgotten to undo this part when push the changes. thx